qutebrowser/browser/downloads.py

```diff
@@ -580,6 +583,9 @@ def on_reply_error(self, code):
         if code == QNetworkReply.OperationCanceledError:
             return
         else:
+            if self.reply is None:
+                log.downloads.debug("QNetworkReply disappeared %s", self)
+                return
```

Comment:
But it happened for you then? Can you reproduce it? If not, I'd prefer to leave this out for now - true, that means more crashes, but that in turn means more information (logs/reports) for me so I can diagnose the real issue.